.github/workflows/workflow.yml

```diff
@@ -151,46 +151,3 @@ jobs:
           directory: gh-pages
           github_token: ${{ secrets.GITHUB_TOKEN }}
           force: true
-
-  publish-to-anaconda:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
```

Comment:
This line was always false in the original workflow so it would skip. Now fixed by moving to a separate file and using a 'on: release' trigger.